Model/Order/PlaceOrder.php

```diff
@@ -540,4 +560,20 @@ private function buildOrderData(array $firstTransaction, int $quoteId, string $p
 
         return $orderData;
     }
+
```

Comment:
Nitpick: methods should only be separated by a single blank line.

Reply:
Done here: e49bc3e2fe84ef7b965545b6232450fd79cf4329